tdrs-backend/tdpservice/users/apps.py

```diff
@@ -8,3 +8,7 @@ class UsersConfig(AppConfig):
 
     name = "tdpservice.users"
     verbose_name = "Users"
+
+    def ready(self):
+        """Import signals."""
+        import tdpservice.users.signals  # noqa
```

Comment:
What is this?

Reply:
I decided to use signals to trigger sending emails.